packages/caliper-fisco-bcos/lib/fiscoBcos.js

```diff
@@ -38,6 +39,12 @@ class FiscoBcos extends BlockchainInterface {
         this.bcType = 'fisco-bcos';
         this.workspaceRoot = workspace_root;
         this.fiscoBcosSettings = CaliperUtils.parseYaml(this.configPath)['fisco-bcos'];
+
+        if (this.fiscoBcosSettings.network && this.fiscoBcosSettings.network.authentication) {
+            for (let k in this.fiscoBcosSettings.network.authentication) {
+                this.fiscoBcosSettings.network.authentication[k] = path.join(this.workspaceRoot, this.fiscoBcosSettings.network.authentication[k]);
```

Comment:
For easily/correctly resolving both absolute and relative paths, use `CaliperUtils.resolvePath(<path>)`:
https://github.com/hyperledger/caliper/blob/master/packages/caliper-core/lib/common/utils/caliper-utils.js#L67

Reply:
You are right, I've modified it.

Reply:
@aklenik